bin/libstd-ecm.sh

```diff
@@ -64,6 +69,11 @@ has_trivy() {
     __cmd_check "${TRIVY}"
 }
 
+has_vagrant() {
+    VAGRANT="$(command -v vagrant)"
+    __cmd_check "${VAGRANT}"
```

Comment:
Can you make sure the Dockerfile is updated to install `vagrant`?

Reply:
This tool will not be run via a Docker container. This is envisioned as a standalone tool,  otherwise we are launching a docker container to then launch VMs within it (not a road I want to go down).